curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md

```diff
@@ -72,7 +72,7 @@ tests:
     }
   </script>
     
-    <script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript">
+    <script async src = "https://docs.opencv.org/master/opencv.js" onload = "onOpenCvReady();" type = "text/javascript">
```

Comment:
Please keep this line unchanged.

Reply:
@ilbalance, May I additionally ask you to remove whitespaces here?

Reply:
Sure !